src/app/fetch-api-data.service.ts

```diff
@@ -25,15 +25,15 @@ export class UserRegistrationService {
   }
 
 // Log in
-login(userDetails: any): Observable<any> {
+userLoginService(userDetails: any): Observable<any> {
```

Comment:
The name of service methods was ok, you don't need to append `Service` at the end of their names. I was suggesting that you change the name of the class `UserRegistrationService`. The class name should be more generic.